gutenberg/blocks/ts/src/components/event-list-inspector-controls.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
```

Comment:
Revert all formatting changes on this file. I will not review this file before doing that because almost every line seems to have changed now.